Fix styles to be compatible with bootstrap & Update demo UI

napei opened the pull request:
This PR makes the control respond to being given a form-control class through bootstrap properly. Previously the width was fixed, and the the control was inline meaning it did not respond like a traditional bootstrap component in a form or in a `form-group` div.

There was also an error in the readme, saying the default value for `cssClass` is "control-form" when it should be "form-control"

Also updated the demo app to use bootstrap styles on buttons, as well as repsonsive layout.

**Before:**
![image](https://user-images.githubusercontent.com/8946502/87499716-400dcc80-c69e-11ea-837b-7940c1f877f6.png)

**After:**
![after](https://user-images.githubusercontent.com/8946502/87499732-4603ad80-c69e-11ea-9561-faba89762164.png)

The traditional bootstrap way of using forms is

```
<form>
  <div class="form-group">
    <label for="phone">Phone number</label>
    <input type="text" class="form-control" name="phone" />
  </div>
</form>
```

Bootstrap form components are `width: 100%` and block level. 
Using the above html format on the default demo app and the input class changed to `form-control` provides the following incorrect format:

![image](https://user-images.githubusercontent.com/8946502/87499988-d7731f80-c69e-11ea-9845-5672fadfd0fa.png)

pasevin commented:
Hey @napei if you still think this PR could be relevant, I will gladly revisit this and merge if you can resolve the conflicts. Otherwise, I will close this.

napei commented:
> Hey @napei if you still think this PR could be relevant, I will gladly revisit this and merge if you can resolve the conflicts. Otherwise, I will close this.

I'll have a look at this in the coming days 😄

napei commented:
@pasevin I've rebased and updated the demo to BS 4.6 (there's an argument to go straight to 5.0 as it'll be released soon)

There's an error I can't figure out, perhaps you can have a look and see what you think.

![image](https://user-images.githubusercontent.com/8946502/110879268-c7e0cf00-8330-11eb-8991-d2f364548fbb.png)
